Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -376,3 +371,64 @@ class GraphLinter: GraphLinting {
 //        ],
     ]
 }
+
+// MARK: - Dependncies linter
+
+extension GraphLinter {
+    /// It verifies setup for packages
+    ///
+    /// - Parameter graph: Project graph.
+    /// - Returns: Linting issues.
+    private func lintPackageDependencies(graph: Graphing) -> [LintingIssue] {
```

Comment:
I'd move these methods to the body of the `GraphLinter` class.

Reply:
I'd rather keep it in the same body and add a annotation to keep swiftlint happy. Unless extensions bring a lot of value, for example grouping methods with the same access level, I'd make things easier for the compile and keep everything under the class definition.